packages/browser/__tests__/ClientAuthentication.spec.ts

```diff
@@ -146,7 +146,7 @@ describe("ClientAuthentication", () => {
         "https://coolapp.com/redirect?state=userId&id_token=idToken&access_token=accessToken";
       await clientAuthn.handleIncomingRedirect(url);
 
-      // Calling handleredirect should give us an authenticated fetch.
+      // Calling handle redirect should give us an authenticated fetch.
```

Comment:
```suggestion
      // Calling `handleRedirect` should give us an authenticated fetch.
```

Reply:
No, I'm trying to fix this. Basically it _seems like_ the original comment was trying to refer to the explicit function name (I think, it wasn't in quotes or anything, and it also wasn't camel-cased, so either way it looked like a typo!), but then the function name seems to have changed (i.e., from `handleRedirect()` to `handleIncomingRedirect()`), at which point the comment got out-of-sync with the code, which is always confusing, and therefore a big reason to try to never refer to things explicitly in comments. So I changed that to just be 'handle redirect' to try and reflect in English the behaviour of the code, but without naming any functions explicitly. Does that make sense now, i.e., we should leave the original change I made)...?

Reply:
I understand the resoning, and it makes sense, but then the phrase "Calling handle redirect" is a bit weird, isn't it ? Couldn't it just be "Handling the redirect" or something ? Anyway, if the sentence is correct in English, I'm happy with your original suggestion.

Reply:
Yeah, I was just trying to tweak the original as little as possible, but I think "Calling the redirect handler..." is better again - I'll change it to that: 9165db5